tor.mk

```diff
@@ -0,0 +1,50 @@
+ifneq ($(PROCURSUS),1)
+$(error Use the main Makefile)
+endif
+
+SUBPROJECTS  += tor
+DOWNLOAD     += https://dist.torproject.org/tor-$(TOR_VERSION).tar.gz
+TOR_VERSION  := 0.4.3.5
+DEB_TOR_V    ?= $(TOR_VERSION)
+
+tor-setup: setup
+	$(call EXTRACT_TAR,tor-$(TOR_VERSION).tar.gz,tor-$(TOR_VERSION),tor)
+
+ifneq ($(wildcard $(BUILD_WORK)/tor/.build_complete),)
+tor:
+	@echo "Using previously built tor."
+else
+tor: tor-setup
+	cd $(BUILD_WORK)/tor && ./configure --prefix=$(BUILD_STAGE)/tor \
```

Comment:
I have no doubt specifying `--prefix=/usr` would alleviate the issue you have during the package stage with everything installing to `$(BUILD_STAGE)/bin` and `$(BUILD_STAGE)/share`.

Reply:
Setting the prefix to /usr will move /etc/tor to /usr/etc/tor I’ll look into how to fix that tonight